.gitignore

```diff
@@ -118,4 +118,6 @@ dist
 .yarn/install-state.gz
 .pnp.*
 # test platform
-test/
\ No newline at end of file
+test/
+
+src/schemaQuery.js
```

Comment:
Not Blocking: how come these are in the `.gitignore` file?

Reply:
will leave for now, should probably add a broader gitignore for js build files